zerver/models.py

```diff
@@ -272,6 +262,8 @@ class Realm(models.Model):
         POLICY_MODERATORS_ONLY,
     ]
 
+    invite_to_realm_policy: int = models.PositiveSmallIntegerField(default=POLICY_MEMBERS_ONLY)
```

Comment:
Maybe we should sort this alphabetically among this block (i.e. after `create_stream_policy`), and also add a descriptive comment?